core/src/main/java/com/novoda/noplayer/internal/exoplayer/forwarder/ExoPlayerForwarder.java

```diff
@@ -58,8 +58,9 @@ public void bind(NoPlayer.CompletionListener completionListener, NoPlayer.StateC
         exoPlayerEventListener.add(new OnCompletionStateChangedForwarder(stateChangedListener));
     }
 
-    public void bind(NoPlayer.ErrorListener errorListener) {
+    public void bind(NoPlayer.ErrorListener errorListener, NoPlayer.ErrorListener internalErrorListener) {
```

Comment:
worth collapsin these params into a list `...`? as public method, you can add a javadoc explaining that the order of the listeners matter

Reply:
That's exactly why I didn't use `varargs` 😄. But maybe I shouldn't even be modifying this and just call the method twice and test that it's called in the right order